src/lib/toQueryString.ts

```diff
@@ -3,7 +3,7 @@ const toQueryString = (params: object): string => {
   return Object.keys(params)
     .filter(
       (key) =>
-        typeof params[key] !== undefined &&
+        typeof params[key] !== "undefined" &&
```

Comment:
is this change related to removing the auto alarm type? judging from the commit history, you plan to squash-merge this, meaning it will end up in a `feat` commit

in projects, this is rarely an issue, but in packages, I'd be very explicit about the commit history. these will result in changelog entries after all. from reading the changelog, this will be an unexpected side effect

Reply:
this is a bug that I need to fix it so I can use this function, I can add it in the feat:

Reply:
allrighty, if it's directly correlated to your code change, let's roll with squashing it

Reply:
but it will change the behavior for other code that consumes this function and this will not be reflected in the changelog now. this decision is ultimately up to you